source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -145,13 +180,13 @@ void TextureBaker::bakeShaderInputs(NodePtr material, NodePtr shader, GenContext
         return;
     }
 
-    std::set<OutputPtr> bakedOutputs;
+    std::unordered_map<OutputPtr, InputPtr> bakedOutputs;
```

Comment:
Now that this container is a map from output to input elements, it might be more clear to name it `bakedOutputMap`.

Reply:
Fixed!